Description: replace ava by tape
Author: Yadd <yadd@debian.org>
Forwarded: not-needed
Last-Update: 2021-12-12

--- a/test.js
+++ b/test.js
@@ -1,5 +1,5 @@
-import test from 'ava';
-import shebangCommand from '.';
+const test = require('tape');
+const shebangCommand = require('.');
 
 test('main', t => {
 	t.is(shebangCommand('#!/usr/bin/env node'), 'node');
@@ -10,4 +10,5 @@
 	t.is(shebangCommand('#!/sh'), 'sh');
 	t.is(shebangCommand('node'), null);
 	t.is(shebangCommand(), null);
+	t.end();
 });
